sklearn/utils/estimator_checks.py

```diff
@@ -513,9 +522,14 @@ def _should_be_skipped_or_marked(estimator, check):
     return False, "placeholder reason that will never be used"
 
 
-def parametrize_with_checks(estimators):
+def parametrize_with_checks(estimators, legacy=True):
```

Comment:
Does this need to be added to `check_estimator`?

Reply:
I'll add it.